docs/guides/1.2.0-upgrade-guide.md

```diff
@@ -0,0 +1,108 @@
+---
+layout: "couchbase-capella"
+page_title: "Couchbase Capella Provider 1.2.0: Upgrade and Information Guide"
+sidebar_current: "docs-couchbase-capella-guides-120-upgrade-guide"
+description: |-
+Couchbase Capella Provider 1.2.0: Upgrade and Information Guide
+---
+
+# Couchbase Capella Provider 1.2.0: Upgrade and Information Guide
+
+Here is a list of what's new in 1.2.0
+
+## New Features
+
+* Create audit log export job with [`couchbase-capella_audit_log_export`](https://registry.terraform.io/providers/couchbasecloud/couchbase-capella/latest/docs/resources/audit_log_export)
+* Create and manage audit log settings with [`couchbase-capella_audit_log_settings`](https://registry.terraform.io/providers/couchbasecloud/couchbase-capella/latest/docs/resources/audit_log_settings)
+* Retrieve audit log event ids for a cluster with data-source [`couchbase-capella_audit_log_event_ids`](https://registry.terraform.io/providers/couchbasecloud/couchbase-capella/latest/docs/data-sources/audit_log_event_ids)
+
+## Changes
+
+There are no deprecations as part of this release.
+
+## Creating Capella Audit Log Export Job
+
+To create a new audit log export job, use the `audit_log_export` resource as follows:
```

Comment:
See Talina's comment elsewhere on not using internal 'job'.

Reply:
It is fine to refer to as a job. We do that in our public Management API as well. It is an export job